.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 100,
```

Comment:
same comment as in the other PR, we need to decide if this is the preferred print width